charmcraft/commands/build.py

```diff
@@ -109,11 +111,38 @@ def _link_to_buildpath(self, srcpath):
         destpath.symlink_to(srcpath)
         return destpath
 
+    def _load_juju_ignore(self):
+        ignore = JujuIgnore(default_juju_ignore)
+        path = self.charmdir / '.jujuignore'
+        if path.exists():
+            with path.open() as ignores:
+                ignore.extend_patterns(ignores)
+        return ignore
+
+    def _walk_unignored(self, topdir):
```

Comment:
Thanks for this.

Anyway, I need to implement the "walk" outside, as it will be ignoring different stuff (a part of which is jujuignored).

So I think this shouldn't be included in this layer.